matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -489,6 +487,6 @@ internal class LocalEchoEventFactory @Inject constructor(
         //     </blockquote>
         // </mx-reply>
         // No whitespace because currently breaks temporary formatted text to Span
-        const val REPLY_PATTERN = """<mx-reply><blockquote><a href="%s">%s</a><a href="%s">%s</a><br />%s</blockquote></mx-reply>%s"""
+        const val REPLY_PATTERN = """<mx-reply><blockquote><a href="%s">In reply to</a><a href="%s">%s</a><br />%s</blockquote></mx-reply>%s"""
```

Comment:
This is not new but I think a space is missing between the two links.

Reply:
I added the space, but is the

> No whitespace because currently breaks temporary formatted text to Span

comment referring to that or something else?

Reply:
No, looking at the history, it seems that before the string was defined on several lines. But I think it's ok like that.

Reply:
(thanks)